### ISSUE TITLE
DOI Manager adjustments

### ISSUE THREAD

**Issue by joemull** [BirkbeckCTP/janeway#2951]
Here are issues users have reported for the DOI Manager.

- [ ] Add confirmation modal to action buttons
- [ ] Move actions to top right of the screen
- [x] The facet titles are currently scrolling with the content which makes it hard to distinguish them. The label should remain fixed.
- [ ] The labels and counts for status are at times confusing. a) There is a bug in which Unknown may appear twice, or some unknown articles are not labelled as such. b) The sum of the facet counts may be less than the total results, leading to questions about the status of the remainder. c) Articles registered outside Janeway are not showing up as registered, so we would need to track the registration status as in #2952. d) We may need to clarify "Unregistered" by splitting it into "No DOI in Janeway" and "DOI not registered by Janeway"
- [ ] Funder information may not appear correctly in the deposit document.
- [x] #2966
- [x] #2967
- [ ] Being able to sort the columns would be very useful. Managing editor: "Being able to sort columns (where possible) is really useful and it's the thing I instinctively look for with lists."
- [ ] The XML buttons open in the same tab, but it may be better to open them in a new tab.
- [ ] The Primary issue facet may be missing some issues, due to an issue not being linked to an article as a primary issue.
- [ ] The Journal facet should probably sort alphabetically.
- [ ] On the press level, the issue labels aren't helpful because it isn't clear which journal they belong to. This might be a reason to filter down the facet choices as facets are selected.
- [ ] Add some screenshots to the documentation.
- [ ] A way to limit the selection of articles for the action buttons. Checkboxes or multi-radio?
- [ ] Make sure all the right stages are excluded in the queryset

**Comment by joemull** [comment]
Backlog refinement: Joe to split these into sub-issues.

**Comment by S-Haime** [comment]
@ajrbyers asked for this to be in urgent triage, to look at some quick improvements for 1.9.1

I think that the main things would be the filters not working properly, followed by that it can be impossible to filter down to a group small enough to work with, depending on issue size!

**Comment by mauromsl** [comment]
BARF Notes:
We are moving this to "Needs Investigation" and then check:

 - Figure out why filtering is no longer working
 - Find a solution for filtering when the number of matched articles is longer than the configured limit (pagination)
 - What are the bottlenecks and how they can be addressed.
 - Create a separate issue for all other problems (including async processing) that can be brought to Janeway 2.0
